core/GlobalState.cc

```diff
@@ -1052,14 +1056,14 @@ SymbolRef GlobalState::enterTypeArgument(Loc loc, SymbolRef owner, NameRef name,
     DEBUG_ONLY(categoryCounterInc("symbols", "type_argument"));
     wasModified_ = true;
 
-    owner.dataAllowingNone(*this)->typeArguments().emplace_back(result);
+    core::SymbolRef(*this, core::SymbolRef::Kind::Method, owner.id())
+        .dataAllowingNone(*this)
```

Comment:
```suggestion
    core::SymbolRef(owner).dataAllowingNone(*this)
```
Shouldn't this work, given the `SymbolRef` constructor that accepts a `MethodRef` argument?

Reply:
You're right. idk why I did this.